e2e-tests/commands/insurance/assert-alternative-currency-form.js

```diff
@@ -9,9 +9,28 @@ import {
 } from '../../fixtures/currencies';
 import checkAutocompleteInput from '../shared-commands/assertions/check-autocomplete-input';
 import { DZA } from '../../fixtures/countries';
+import partials from '../../partials';
 
 const { CURRENCY: { CURRENCY_CODE, ALTERNATIVE_CURRENCY_CODE } } = INSURANCE_FIELD_IDS;
 
+/**
+ * returns field for currency/alternative currency
+ * if currency provided, then returns radio for respective currency option
+ * if no currency provided, then returns radio for alternative currency
+ * @param {String} FIELD_ID - fieldId for currency radio
+ * @param {String} ALTERNATIVE_CURRENCY_FIELD_ID - fieldId for alternative currency radio
+ * @param {String} currency - Currency string
+ */
+const whichRadio = (FIELD_ID, ALTERNATIVE_CURRENCY_FIELD_ID, currency) => {
```

Comment:
Can we rename `whichRadio`? Too generic, also can we have camel case params here? Otherwise look like constants

Also need documentation for `@return`